runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -580,6 +580,20 @@ void __quantum__qis__MultiRZ(double theta, bool adjoint, int64_t numQubits, ...)
                                                              /* inverse = */ adjoint);
 }
 
+void __quantum__qis__ISWAP(QUBIT *wire0, QUBIT *wire1, bool adjoint)
+{
+    Catalyst::Runtime::getQuantumDevicePtr()->NamedOperation(
+        "ISWAP", {}, {reinterpret_cast<QubitIdType>(wire0), reinterpret_cast<QubitIdType>(wire1)},
```

Comment:
Why do we need these `reinterpret_cast`? Is there any way that we can rewrite this code without casting?

Reply:
Isn't that what we are doing for other gates as well?

Reply:
just wanted to know @paul0403's thoughts on this :)

Reply:
Nice insight!

Reply:
Awesome! :guitar: